src/projetos/HomeProjetosListPage.tsx

```
@@ -184,10 +184,11 @@ export function HomeProjetosListPage() {
               placeholder="Buscar projeto..."
               value={search}
               onChange={(e) => setSearch(e.target.value)}
-              className="bg-white rounded-full pr-10" // espaço pro ícone
+              className="bg-white rounded-full" // espaço pro ícone
             />
-            <CiSearch className="absolute right-3 top-1/2 -translate-y-1/2 text-xl text-gray-500 pointer-events-none" />
           </div>
+             <FiltroProjetos onFiltrar={(resultados) => setProjetos(resultados)} />
+
```

ESLint

[error] Replace `···<FiltroProjetos·onFiltrar={(resultados)·=>·setProjetos(resultados)}·/>⏎` with `<FiltroProjetos·onFiltrar={(resultados)·=>·setProjetos(resultados)}·/>`

(prettier/prettier)